lib/prometheus_exporter/middleware.rb

```diff
@@ -58,6 +58,13 @@ def default_labels(env, result)
     if params
       action = params["action"]
       controller = params["controller"]
+    elsif env["rack.cors"] && env["rack.cors"].preflight?
```

Comment:
maybe we check that it responds to `preflight?` just in case?

Reply:
Good idea. I've added a `respond_to?` for this and also abstained from using the `REQUEST_PATH` as action, since that could have a huge impact on the cardinality of the metrics.